sql/core/src/test/resources/tpcds-query-results/v1_4/q23b.sql.out

```diff
@@ -3,7 +3,4 @@
 -- !query schema
 struct<c_last_name:string,c_first_name:string,sales:decimal(28,2)>
 -- !query output
-NULL	Robert              	598.86
-Brown                         	Monika              	6031.52
-Collins                       	Gordon              	727.57
-Green                         	Jesse               	9672.96
+
```

Comment:
Does the empty result set look right?

Reply:
Yea, I need to check the answers query-by-query later...